filter_broken_pixels: true

filter_sigma: 2.0
